kolibri/plugins/learn/assets/src/state/prepareLearnApp.js

```diff
@@ -0,0 +1,22 @@
+const { currentUserId } = require('kolibri.coreVue.vuex.getters');
+const { MembershipResource } = require('kolibri').resources;
+
+// prepares state that is used for all pages in 'learn' plugin/app
+// currently, this is only the user's memberships
+function prepareLearnApp(store) {
+  const userId = currentUserId(store.state);
+
+  if (userId === null) return Promise.resolve();
+
+  const membershipPromise = MembershipResource.getCollection({ user_id: userId }).fetch();
```

Comment:
We should bootstrap this kind of data into django template, so that this fetch resolves immediately.